npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-rate-limit",
```

Comment:
Since this is a plugin that will be used by other modules, you shouldn't have a shrinkwrap file. We should allow whatever versions are compatible instead of locking it down.